### DISCUSSION TITLE
v1.39.4

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1630] [General]
## Fixes

This release fixes the following:

- Fixed issue with assigning owner to website, #1614 
- Fixed issue with deleting accounts, #1624 

## Updates
- Updated Nextjs to `v12.3.2`
- Language translation updates

A huge thanks to all the contributors on this release! @aidanm1999 @RikoDEV @enricopaulini @briancao 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v1.39.4'>v1.39.4</a>.</em>

**Comment by andlil** [comment]
Trying https://docker.umami.is/umami-software/umami:postgresql-latest gives 502 Bad Gateway, perhaps you could have a look at it @mikecao

**Reply by eallion** [reply]
same

**Reply by mikecao** [reply]
Should be working now.

**Reply by tombrereton** [reply]
Still getting 502 (3rd November). I raised an Issue before reading this - let me know if you want me to close it.

https://github.com/umami-software/umami/issues/1634
